browser/src/canvas/sections/CommentListSection.ts

```diff
@@ -2061,12 +2062,17 @@ export class CommentSection extends CanvasSectionObject {
 			this.resizeComments();
 
 		lastY += app.activeDocument.activeView.viewedRectangle.pY1;
+
+		let horizontalScroll = app.activeDocument.fileSize.x;
+		if (availableSpace < this.sectionProperties.commentWidth && !this.isCollapsed)
+			horizontalScroll = (app.activeDocument.fileSize.pX + this.sectionProperties.commentWidth) * app.pixelsToTwips;
```

Comment:
Since commentWidth is in CSS pixels, we need to use app.activeDocument.fileSize.**c**X here. Also we need to multiply with app.dpiScale along with app.pixelsToTwips.

If dpiScale is equal to 1, you cannot see the difference, but there should be when dpiScale is different than 1.